src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Startup.cs

```diff
@@ -63,14 +63,23 @@ public void ConfigureServices(IServiceCollection services)
 
         // MVC
 
+        services.AddRequestTimeouts(options =>
+        {
+            options.DefaultPolicy =
+                new RequestTimeoutPolicy
+                {
+                    Timeout = TimeSpan.FromMilliseconds(30000),
```

Comment:
I don't knwo if it's worth making this configurable (with a 30s fallback), and set to something like 500ms in our integration tests?  That way, we can register a test controller that delays for 1s and then we can test the response.

Reply:
Good idea, I'll make that change!